panda/html/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="kor">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="icon" href="../img/randing/favicon.ico">
+    <link href="https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff">
+</head>
+<body>
+    <header>
+        <div class="header-top">
+            <a href="../html/index.html">
+                <img class="logo" src="../img/randing/logoimg.png">
+            </a>
+            <a class="btn" href="../html/login.html">로그인</a>
+        </div>
+        <div class="header-bg">
+            <h1>일상의 모든 물건을<br>거래해 보세요</h1>
+            <a class="btn" href= "items.html">구경하러 가기</a>
+        </div>
+    </header>
+    <main>
+        <section>
+            <div class="tab">
+                <img src="../img/randing/home_1.png" alt="hot item">
+                <div  class="tab-txt">
+                    <h5>Hot item</h5>
+                    <h1>인기 상품을 <br> 확인해 보세요</h1>
+                    <p>
+                        가장 HOT한 중고거래 물품을<br>
+                        판다마켓에서 확인해 보세요
+                    </p>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="tab reverse">
+                <img src="../img/randing/home_2.png" alt="search">
+                <div  class="tab-txt">
+                    <h5>Serach</h5>
+                    <h1>구매를 원하는 <br>상품을 검색하세요</h1>
+                    <p>
+                        구매하고 싶은 물품을 검색해서<br>
+                       쉽게 찾아보세요
+                    </p>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="tab">
+                <img src="../img/randing/home_3.png" alt="register">
+                <div  class="tab-txt">
+                    <h5>Resister</h5>
+                    <h1> 판매를 원하는<br>상품을 등록하세요</h1>
+                    <p>
+                        어떤 물건이든 판매하고 싶은 상품을<br>
+                        쉽게 등록하세요
+                    </p>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-bg">
+            <h1>
+                믿을 수 있는<br>
+                판다마켓 중고거래
+            </h1>
```

Comment:
`h1` 태그는 가장 중요한 헤드라인에만 사용합니다. 
```suggestion
            <p>
                믿을 수 있는<br>
                판다마켓 중고거래
            </p>
```